pyscript.core/src/stdlib/pyweb/base.py

```diff
@@ -0,0 +1,30 @@
+import inspect
+
+import js
```

Comment:
this is not what we should import in the stdlib unless you want this to throw in workers ... please see display.py imports

Reply:
This whole code is meant to be removed because we don't want to reinvent the `when` from `pyscript`. It was a placeholder to allow me to keep experimenting while we bring it back